website/content/docs/internals/telemetry.mdx

```diff
@@ -377,54 +377,72 @@ These metrics relate to the supported [storage backends][storage-backends].
+| `vault.raft_storage.bolt.transaction.`<br/>`started_read_transactions`        | Number of started read transactions.                                                                                                                                                                       | transactions                      | gauge  |
+| `vault.raft_storage.bolt.transaction.`<br/>`currently_open_read_transactions` | Number of currently open read transactions.                                                                                                                                                                | transactions                      | gauge  |
+| `vault.raft_storage.bolt.page.count`                                   | Number of page allocations.                                                                                                                                                                                       | allocations                       | gauge  |
+| `vault.raft_storage.bolt.page.`<br/>`bytes_allocated`                         | Total bytes allocated.                                                                                                                                                                                     | bytes                             | gauge  |
+| `vault.raft_storage.bolt.cursor.count`                                 | Number of cursors created.                                                                                                                                                                                        | cursors                           | gauge  |
+| `vault.raft_storage.bolt.node.count`                                   | Number of node allocations.                                                                                                                                                                                       | nodes                             | gauge  |
+| `vault.raft_storage.bolt.node.dereferences`                            | Number of node dereferences.                                                                                                                                                                                      | dereferences                      | gauge  |
+| `vault.raft_storage.bolt.rebalance.count`                              | Number of node rebalances.                                                                                                                                                                                        | rebalances                        | gauge  |
+| `vault.raft_storage.bolt.rebalance.time`                               | Total time spent rebalancing.                                                                                                                                                                                     | ms                                | sample  |
```

Comment:
I prefer the phrasing we use for time-based samples elsewhere on this page "time taken to" vs "total time".  "Total time" for me would imply an ever-growing counter.

Reply:
ecfc7d3c2de6c04f68d41aa7efca4dc30a9a5132